services/job.h

```diff
@@ -132,22 +146,71 @@ class CompileJob
 
     void setOutputFile(const std::string &file)
     {
-        m_output_file = file;
+        m_output_files[0] = file;
     }
 
     std::string outputFile() const
     {
-        return m_output_file;
+        return m_output_files.at(0);
     }
 
-    void setDwarfFissionEnabled(bool flag)
+    const OutputFiles& outputFiles() const
+    {
+        return m_output_files;
+    }
+
+    void setExtraOutputFile(uint32_t index, const std::string &file)
+    {
+        //printf("setExtraOutputFile[%d] = %s\n", index, file.c_str());
```

Comment:
dead code, please remove, or use the standard logging means to log this

Reply:
will do these, just been away